### ISSUE TITLE
Not working with package.reload

### ISSUE THREAD

**Issue by TvoroG** [tarantool/metrics#121]
`metrics == 0.5.0`

```
unix/:/var/run/tarantool/app.control> package.reload()
---
- error: '/.rocks/share/tarantool/metrics/quantile.lua:10: cannot change a protected
    metatable'
...
```

**Comment by ZolotarevIgor-mail-ru** [comment]
@TvoroG 
Fixed in #149. Fix is already available in scm and will be included in the next release.

**Comment by TvoroG** [comment]
Thanks a lot! Will update soon.
